docs/source/markdown/podman-system-service.1.md

```diff
@@ -17,12 +17,14 @@ The REST API provided by **podman system service** is split into two parts: a co
 Documentation for the latter is available at *https://docs.podman.io/en/latest/_static/api.html*.
 Both APIs are versioned, but the server will not reject requests with an unsupported version set.
 
+Note: To provide additional information about remote requests **podman system service** always updates the log-level option to at least *info*.
```

Comment:
I think this should be a recommends and allow the user to flexibility to set it anyway they want.